backend/de.metas.salescandidate.base/src/main/java/de/metas/ordercandidate/process/ProcessOLCands.java

```diff
@@ -20,10 +23,11 @@
  * The actual work is done by {@link IOLCandBL#process(OLCandProcessorDescriptor, AsyncBatchId)}
  *
  * @author metas-dev <dev@metasfresh.com>
- *
  */
 public class ProcessOLCands extends JavaProcess
```

Comment:
please rename it to something like `C_OLCandEnqueueForSalesOrderCreation` (boyscout-rule - start process name with related table name)